world_cup/lib/player.rb

```diff
@@ -0,0 +1,13 @@
+
+class Player
```

Comment:
This class looks good. I would have liked to see you work through some of iteration 2.